packages/components/src/ResourceList/ResourceList.component.js

```diff
@@ -26,12 +26,14 @@ function ResourceList({
 	onRowClick,
 	renderAs,
 	toolbar,
+	options,
```

Comment:
Maybe we could generalise a bit the name passed to the rowSelectionRenderer like
```suggestion
	rowProps,
```